docs/MACOS.md

```diff
@@ -0,0 +1,17 @@
+# Mezmo Agent on MACOS
```

Comment:
macOS - to please mac funs? here and in other places below.

Reply:
updated